backend/controller/sql/queries.sql

```diff
@@ -384,4 +384,4 @@ INSERT INTO events (deployment_id, request_id, type,
                     custom_key_1, custom_key_2, custom_key_3, custom_key_4,
                     payload)
 VALUES ($1, $2, $3, $4, $5, $6, $7, $8)
-RETURNING id;
\ No newline at end of file
+RETURNING id;
```

Comment:
What's the purpose of this? It will return the ID of the last row no?

Reply:
I think this is just a weird github format. The change is that I removed the newline at the end of the file (I'll fix that) so this `RETURNING id;` is for `InsertEvent` and not related to this change. Sorry for the confusion.